server/test/unit/com/thoughtworks/go/server/service/PipelineConfigServiceTest.java

```diff
@@ -180,4 +183,19 @@ public void createPipelineConfigShouldValidateAllPluggableTasks() {
         verify(pluggableTaskService).isValid(xUnit);
         verify(pluggableTaskService).isValid(docker);
     }
+
+    @Test
+    public void shouldGetPipelinesCount() {
+        assertThat(pipelineConfigService.totalPipelinesCount(), is(this.cruiseConfig.allPipelines().size()));
+    }
+
+    @Test
+    public void pipelineCountShouldIncludeConfigRepoPipelinesAsWell() {
```

Comment:
Will the number of pipelines shown in `/go/api/support` be consistent with this number? Meaning, does `/go/api/support` also include config repo pipelines?

Reply:
yes it does